pkg/config/remote/api/http.go

```diff
@@ -37,13 +37,13 @@ type HTTPClient struct {
 }
 
 // NewHTTPClient returns a new HTTP configuration client
-func NewHTTPClient(apiKey, appKey string) *HTTPClient {
+func NewHTTPClient(authHeaders map[string]string) *HTTPClient {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏Using a struct and using that here would probably be a bit more secure, but it's not that big of a deal